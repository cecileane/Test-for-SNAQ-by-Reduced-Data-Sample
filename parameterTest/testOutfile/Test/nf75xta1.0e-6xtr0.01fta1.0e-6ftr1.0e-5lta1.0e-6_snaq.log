optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=1.0e-6,
                       xtolAbs=1.0e-6, xtolRel=0.01.
 max number of failed proposals = 75, liktolAbs = 1.0e-6.
rootname for files: nf75xta1.0e-6xtr0.01fta1.0e-6ftr1.0e-5lta1.0e-6_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 78741
seed: 78741 for run 1
Sat Oct  1 17:11:23 2016

 BEGIN SNaQ for run 1, seed 78741 and hmax 1
Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 152 (failures: 75) with -loglik=127.5115 and ht_min=[0.45392,0.0,0.5186,2.23709,10.0,0.0,0.36548]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 43		 43		 5	 |	 0		 0		 --
mvtarget	 38		 38		 1	 |	 1		 1		 --
chdir		 14		 14		 0	 |	 1		 1		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 56		 38		 0	 |	 1		 1		 --
Total		 152		 134		 7	 |	 3		 3		 2
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 0.7

 FINISHED SNaQ for run 1, -loglik of best 127.51150018526096
(2,((6,(5,#H7:0.36547543533805604::0.4539155300841907):10.0):2.2370930848598465,1):0.5186001312829344,((4,3):0.0)#H7:0.0::0.5460844699158093);
---------------------
seed: 78400 for run 2
Sat Oct  1 17:11:37 2016

 BEGIN SNaQ for run 2, seed 78400 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 112 (failures: 75) with -loglik=126.74251 and ht_min=[0.2856,10.0,1.51213,0.51916,0.48942]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 32		 32		 0	 |	 0		 0		 --
mvtarget	 31		 31		 1	 |	 0		 0		 --
chdir		 11		 11		 0	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 37		 27		 3	 |	 0		 0		 --
Total		 112		 102		 5	 |	 0		 0		 0
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 0.0		 0.0

 FINISHED SNaQ for run 2, -loglik of best 126.74250981975025
((2,(1)#H7:::0.7143969533046639):1.512133721249782,4,(((#H7:::0.28560304669533604,6):10.0,5):0.5191608104492336,3):0.48941877961877683);
---------------------

Sat Oct  1 17:11:46 2016
MaxNet is ((2,(1)#H7:::0.7143969533046639):1.512133721249782,4,(((#H7:::0.28560304669533604,6):10.0,5):0.5191608104492336,3):0.48941877961877683); 
with -loglik 126.74250981975025
