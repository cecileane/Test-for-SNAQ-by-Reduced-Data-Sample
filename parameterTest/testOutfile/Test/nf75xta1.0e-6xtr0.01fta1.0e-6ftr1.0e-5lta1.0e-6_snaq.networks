((2,(1)#H7:::0.7143969533046639):1.512133721249782,4,(((#H7:::0.28560304669533604,6):10.0,5):0.5191608104492336,3):0.48941877961877683);, with -loglik 126.74250981975025 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
((2,(#H7:::0.3873849170887209,1):1.0745415979707174):0.6805452771154034,4,(((6)#H7:::0.6126150829112791,5):4.436047769911434,3):0.48737651290398637);, with -loglik 131.23956405285108
(#H7:::0.4169651708278178,4,((((1,(2)#H7:::0.5830348291721822):3.6595166026431167,6):0.13575891007894214,5):0.06415167761518954,3):1.7757236734020103);, with -loglik 192.91920817571685
((4)#H7:::0.5165600203239782,2,((((3,#H7:::0.48343997967602176):0.09889935942663157,5):0.25028780249141913,6):1.073021399738462,1):0.19002936112955937);, with -loglik 205.18581127216427
((2,(((#H7:::0.4039699137222278,5):9.554746838748937,6):1.0022453809860097,1):0.09786859159376418):0.07497814502901666,4,(3)#H7:::0.5960300862777722);, with -loglik 215.1316103101926
((2,((#H7:::0.4650982946256489,6):5.162813020441703,1):0.04473755926890497):0.21757183618653841,4,((5)#H7:::0.5349017053743511,3):0.997360246575831);, with -loglik 232.1859503431418
