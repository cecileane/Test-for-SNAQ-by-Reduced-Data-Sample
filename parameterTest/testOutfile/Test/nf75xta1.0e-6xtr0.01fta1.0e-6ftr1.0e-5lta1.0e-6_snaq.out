((2,(1)#H7:::0.7143969533046639):1.512133721249782,4,(((#H7:::0.28560304669533604,6):10.0,5):0.5191608104492336,3):0.48941877961877683); -Ploglik = 126.74250981975025
 Dendroscope: ((2,(1)#H7):1.512133721249782,4,(((#H7,6):10.0,5):0.5191608104492336,3):0.48941877961877683);
 Elapsed time: 22.979128387 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 ((2,(1)#H7:::0.7143969533046639):1.512133721249782,4,(((#H7:::0.28560304669533604,6):10.0,5):0.5191608104492336,3):0.48941877961877683);, with -loglik 126.74250981975025
 (2,((6,(5,#H7:0.36547543533805604::0.4539155300841907):10.0):2.2370930848598465,1):0.5186001312829344,((4,3):0.0)#H7:0.0::0.5460844699158093);, with -loglik 127.51150018526096
-------
